src/ScrollBar.tsx

```diff
@@ -193,6 +193,9 @@ const ScrollBar = React.forwardRef<ScrollBarRef, ScrollBarProps>((props, ref) =>
 
   React.useEffect(() => {
     delayHidden();
+     return () => {
+      clearTimeout(visibleTimeoutRef.current);
```

Comment:
`clearTimeout` already exists in `delayHidden` function.

Reply:
When destroying components, scheduled tasks should be cleaned up, otherwise destroying components will result in an error message

Reply:
You are right.